kubelet-to-gcm/monitor/main/daemon.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	log "github.com/golang/glog"
+	flag "github.com/spf13/pflag"
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+
+	v3 "google.golang.org/api/monitoring/v3"
+	"k8s.io/contrib/kubelet-to-gcm/monitor"
+)
+
+const (
+	missingValue = "/MISSING"
+	scope        = "https://www.googleapis.com/auth/monitoring.write"
+	testPath     = "https://test-monitoring.sandbox.googleapis.com"
+)
+
+var (
+	// Flags to identify the Kubelet.
+	zone    = flag.String("zone", "us-central1-b", "The zone where this kubelet lives.")
```

Comment:
Using the metadata server isn't an option when you're hitting a remote kubelet. Since I'm no longer doing that, I suppose I could hit get it programmatically.